dom/nodes/getElementsByClassName-08.htm

```diff
@@ -8,8 +8,8 @@
  <body class="a">
   <div id="log"></div>
   <script> test(function() {
-                  document.getElementsByClassName("a\fa"), [document.body]
-                })
+                  assert_array_equals(document.getElementsByClassName("a\fa"), [document.body]
+  );})
```

Comment:
I should have noted this sooner, but would you be willing to tidy up the indentation a bit up as well?